examples/LM_Bundle_Adjust.ipynb

```diff
@@ -0,0 +1,725 @@
+      "iteration 4: sigma0 = 0.38344720841227964\n",
+      "\n",
+      "corrections: mean = -3.4431393648009435e-05 min = -0.10353090458980926 max = 0.12256234990803173\n",
+      "iteration 5: sigma0 = 0.3834009546378761\n",
+      "\n",
+      "corrections: mean = -7.874567579582659e-07 min = -0.047459113242319884 max = 0.04811283437216138\n",
+      "iteration 6: sigma0 = 0.38336874896690964\n",
+      "\n",
+      "corrections: mean = -1.0811607997045452e-07 min = -0.006589208230282579 max = 0.006256017754636698\n",
+      "iteration 7: sigma0 = 0.38336379718944824\n",
+      "\n",
+      "corrections: mean = -1.2410151661434289e-08 min = -0.0006846136434108924 max = 0.0006444490294294532\n",
+      "iteration 8: sigma0 = 0.3833634726322458\n",
+      "\n",
+      "corrections: mean = -1.2621809061683805e-09 min = -6.87286093893794e-05 max = 6.46395672752282e-05\n",
+      "iteration 9: sigma0 = 0.3833634435849249\n",
+      "\n",
+      "corrections: mean = -1.2719068762773128e-10 min = -6.875558894845864e-06 max = 6.46501388847706e-06\n",
+      "iteration 10: sigma0 = 0.38336344071468026\n",
+      "\n",
+      "corrections: mean = -1.095976326404826e-11 min = -6.865779050542945e-07 max = 6.472822250495515e-07\n",
+      "iteration 11: sigma0 = 0.38336344042918163\n",
+      "\n",
+      "corrections: mean = 3.373646625496664e-13 min = -6.901584048236878e-08 max = 6.471173653807224e-08\n",
+      "iteration 12: sigma0 = 0.3833634404029933\n",
+      "\n",
+      "change in sigma0 of 2.618832928291681e-11 converged!\n"
+     ]
+    }
+   ],
+   "source": [
+    "## sensors = generate_sensors(cubes) # generate sensors\n",
+    "cnet = io_controlnetwork.from_isis(network) # load in network\n",
+    "cnet = compute_apriori_ground_points(cnet, sensors) # calculate ground points\n",
+    "\n",
+    "### INPUTS ###\n",
+    "all_parameters = {sn: get_sensor_parameters(sensor) for sn, sensor in sensors.items()} #all parameters\n",
+    "parameters = {sn: parameter[6:12] for sn, parameter in all_parameters.items()} #just solving for camera angles and angle velocity\n",
+    "##############\n",
+    "\n",
+    "column_dict = compute_coefficient_columns(cnet, sensors, parameters)\n",
+    "num_parameters = max(col_range[1] for col_range in column_dict.values())\n",
+    "num_observations = 2 * len(cnet)\n",
+    "W_observations = np.eye(num_observations)\n",
+    "W_params = compute_parameter_weights(cnet, sensors, parameters, column_dict)\n",
+    "\n",
+    "iteration = 0\n",
+    "V = compute_residuals(cnet, sensors)\n",
+    "dX = np.zeros(W_params.shape[0]) #initialize for sigma calculatio\n",
+    "sigma0 = compute_sigma(V, dX, W_params, W_observations)\n",
+    "print(f'iteration {iteration}: sigma0 = {sigma0}\\n')\n",
+    "\n",
+    "max_iterations = 100\n",
+    "tol = 1e-10\n",
+    "total_correction = np.zeros(num_parameters)\n",
+    "damping = 100\n",
```

Comment:
Earlier you state that the Numerical Recipies's heuristic was damping = 0.01 and I am not seeing 1/damping in the Normal Equation below.

Reply:
Oops. I was playing around with different values to see how it affected the convergance and clearly forgot to revert this before PRing in. Thanks for noticing this!